x-pack/plugins/files/server/saved_objects/file.ts

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { SavedObjectsType } from '@kbn/core/server';
+import { FILE_SO_TYPE } from '../../common';
+import type { FileSavedObjectAttributes } from '../../common';
+
+export const fileObjectType: SavedObjectsType<FileSavedObjectAttributes> = {
+  name: FILE_SO_TYPE,
+  hidden: true,
+  namespaceType: 'multiple-isolated',
+  mappings: {
+    dynamic: false,
+    properties: {
+      created_at: {
+        type: 'date',
+      },
+      updated_at: {
+        type: 'date',
+      },
+      name: {
+        type: 'keyword',
```

Comment:
Maybe this could be text and we could have another `alt` field, where we could store more human readable text (like used as `alt` tag in images):

```suggestion
        type: 'text',
      },
      alt: {type: 'text'},
```